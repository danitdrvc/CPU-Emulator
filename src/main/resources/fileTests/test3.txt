MUL QWORD R0, R1
ADD QWORD R2, R1
DIV QWORD R3, R0
HALT