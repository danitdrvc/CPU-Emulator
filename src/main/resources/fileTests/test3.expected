[50, 5, 5, 0]